### DISCUSSION TITLE
Difficulties to load historical data

### DISCUSSION THREAD

**Discussion by nmatton** [xoolive/traffic#374] [Q&A]
Hi there !
I try to load historical data but the process is going veeery slow, and most of the time ends up to be stuck.
Here is a small portion of code that reproduce the issue : 
```pyhon
from traffic.data import opensky
from traffic.data import eurofirs
ebbu = eurofirs['EBBU']
flights = opensky.history(
    start='2023-07-15 00:00'
    airport = "EBBR",
    bounds=ebbu
)
```

I usually get the 2 or 3 first iterations but it then get stucks, or take a very long time (>10 minutes) per next iteration.

Is it an issue of the opensky service or I misuse it somewhere ?

Thanks for your help !

**Comment by xoolive** [comment]
follow up in #375
